src/Lunr/Ticks/PHPMailer/PHPMailer.php

```diff
@@ -0,0 +1,281 @@
+     * @param string                                        $from    Email address of sender
+     * @param array{smtp_transaction_id?: bool|string|null} $extra   Extra information of possible use
+     *
+     * @return void
+     */
+    protected function afterSending(bool $isSent, array $to, array $cc, array $bcc, string $subject, string $body, string $from, array $extra): void
+    {
+        if ($this->analyticsDetailLevel === AnalyticsDetailLevel::None)
+        {
+            return;
+        }
+
+        $endTimestamp = microtime(TRUE);
+
+        $fields = [
+            'startTimestamp' => $this->startTimestamp,
+            'endTimestamp'   => $endTimestamp,
+            'executionTime'  => (float) bcsub((string) $endTimestamp, (string) $this->startTimestamp, 4),
+            'url'            => $this->Host, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            'ip'             => gethostbyname($this->Host), // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            'traceID'        => $this->tracingController->getTraceId(),
+            'spanID'         => $this->tracingController->getSpanId(),
+            'parentSpanID'   => $this->tracingController->getParentSpanId(),
+        ];
+
+        if ($this->analyticsDetailLevel->atleast(AnalyticsDetailLevel::Detailed))
+        {
+            $options = [
+                'from' => $from,
+                'to'   => $to,
+                'cc'   => $cc,
+                'bcc'  => $bcc,
+            ];
+
+            // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            if ($this->Mailer === 'smtp')
+            {
+                $options += [
+                    'SMTPHost'      => $this->Host, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPPort'      => $this->Port, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPHelo'      => $this->Helo, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPSecure'    => $this->SMTPSecure,
+                    'SMTPAutoTLS'   => $this->SMTPAutoTLS,
+                    'SMTPAuth'      => $this->SMTPAuth,
+                    'SMTPUsername'  => $this->Username, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPPassword'  => $this->Password, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPKeepAlive' => $this->SMTPKeepAlive,
+                    'SMTPAuthType'  => $this->AuthType, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPTimeout'   => $this->Timeout, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                ];
+
+                $options += $this->SMTPOptions;
+            }
+
+            $options = json_encode($options + $extra);
+
+            $fields['requestHeaders'] = $this->prepareLogData($this->MIMEHeader);
```

Comment:
This is a string in Email format. That's not very user-friendly for the analytics, since you get no formatting help whatsoever. Finding a specific header in the list is going to be very annoying.
You'd want to get an array representation (key->value) of this, which we can JSON encode